_posts/2023-04-19-Announcing-Data-Prepper-2.2.0.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: post
+title:  "Announcing Data Prepper 2.2.0"
+authors:
+- dlv
+date: 2023-04-19 14:30:00 -0500
+categories:
+  - releases
+meta_keywords: Data Prepper 2.1.0
+meta_description: Data Prepper 2.1.0 improves data delivery assurances with end-to-end acknowledgements and an S3 DLQ.
+---
+
+Data Prepper 2.2.0 is now available for [download](https://opensearch.org/downloads.html#data-prepper)!
+This release has a number of changes that help with Data Prepper’s reliability and data delivery assurances.
+
+## S3-based dead-letter queue for OpenSearch
+
+Data Prepper’s `opensearch` sink can now write failed documents to an Amazon S3 dead-letter queue (DLQ) to help with analyzing failures. 
```

Comment:
"Prior to Data Prepper 2.2.0, the `opensearch` sink could only write failed events to a local file. This required logging into your cloud instance or machine to retrieve failed events as well creating a different infrastructure to export them.

Now, the `opensearch` sink can write documents from failed events directly in to an Amazon S3 dead-letter- queue (DLQ). This will help you analyze event failures without having to retrieve them locally. Furthermore, users that run on a serverless infrastructure can avoid maintaining a persistent fail state on serverless machines."